test/e2e/persistent_volumes-disruptive.go

```diff
@@ -222,7 +222,7 @@ func initTestCase(f *framework.Framework, c clientset.Interface, pvConfig persis
 
 // tearDownTestCase destroy resources created by initTestCase.
 func tearDownTestCase(c clientset.Interface, f *framework.Framework, ns string, pod *v1.Pod, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) {
-	deletePod(f, c, ns, pod)
```

Comment:
Yes. The pod's ns is the same as used by the test and is set in the pod -- see `makePod()`